mlrun/utils/helpers.py

```diff
@@ -27,8 +27,9 @@
 from tabulate import tabulate
 from yaml.representer import RepresenterError
 
-from ..config import config
+import mlrun.utils.version
```

Comment:
```suggestion
import ..version
```
Would resolve the issue with circular imports?

Reply:
Nope, the problem is that almost no matter what you will import in the code, it will import config.
so Config can't import anything from the package

Reply:
Got it